geomesa-convert/geomesa-convert-common/src/test/scala/org/locationtech/geomesa/convert2/transforms/ExpressionTest.scala

```diff
@@ -231,17 +231,17 @@ class ExpressionTest extends Specification {
         exp.eval(Array("", "2015-01-01T00:00:00.000Z")).asInstanceOf[Date] must be equalTo testDate
       }
     }
-    "parse isodate" >> {
-      val exp = Expression("isodate($1)")
-      exp.eval(Array("", "20150101")).asInstanceOf[Date] must be equalTo testDate
+    "parse isoDate" >> {
+      val exp = Expression("isoDate($1)")
+      exp.eval(Array("", "2015-01-01")).asInstanceOf[Date] must be equalTo testDate
     }
     "parse basicDate" >> {
       val exp = Expression("basicDate($1)")
       exp.eval(Array("", "20150101")).asInstanceOf[Date] must be equalTo testDate
     }
-    "parse isodatetime" >> {
-      val exp = Expression("isodatetime($1)")
-      exp.eval(Array("", "20150101T000000.000Z")).asInstanceOf[Date] must be equalTo testDate
```

Comment:
Is it worth adding unit tests that show that these strings ("20150101", "20150101T000000.000Z") work with basicDate and basicDateTime?

Reply:
i think those are already there, like the one right above this on line 239-240

Reply:
Cool.  Merging.